packages/server/server/api/trecipe/trecipe.interface.ts

```diff
@@ -0,0 +1,15 @@
+interface DestWithStatus {
+    destUUID: string;
+    completed: boolean;
+}
+
+export default interface Trecipe {
+    uuid: string;
+    name: string;
+    description: string;
+    owner: string;
+    isPrivate: boolean;
+    collaborators: Array<string>;
+    image: string;
```

Comment:
Do we need img thumbnail?

Reply:
What's the thumbnail for?? Are you referring to the smaller images on Trecipe cards vs the bigger images in Trecipe page?